Apps/Examples/Examples/All Examples/ShowHideLayerExample.swift

```diff
@@ -0,0 +1,164 @@
+        // is on. `visibility` is `nil` by default.
+        do {
+            try style.updateLayer(withId: museumLayerId) { (layer: inout CircleLayer) in
+                layer.visibility = .constant(sender.isOn ? .visible : .none)
+            }
+        } catch {
+            print("Failed to update the visibility for layer with id \(museumLayerId). Error: \(error.localizedDescription)")
+        }
+    }
+
+    @objc func toggleContourLayerVisibility(sender: UISwitch) {
+        let style = mapView.mapboxMap.style
+        // Update the contour layer's visibility based on whether the switch
+        // is on.
+        do {
+            try style.updateLayer(withId: contourLayerId) { (layer: inout CircleLayer) in
+                layer.visibility = .constant(sender.isOn ? .visible : .none)
+            }
+        } catch {
+            print("Failed to update the visibility for layer with id \(contourLayerId). Error: \(error.localizedDescription)")
+        }
+    }
+
+    func addVisibilitySwitches() {
+        // Create switches to toggle the layers' visibility.
+        let museumSwitch = UISwitch()
+        museumSwitch.addTarget(self, action: #selector(toggleMuseumLayerVisibility(sender:)), for: .valueChanged)
+        museumSwitch.isOn = true
+        museumSwitch.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(museumSwitch, aboveSubview: mapView)
+
+        let contourSwitch = UISwitch()
+        contourSwitch.addTarget(self, action: #selector(toggleContourLayerVisibility(sender:)), for: .valueChanged)
+        contourSwitch.isOn = true
+        contourSwitch.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(contourSwitch, aboveSubview: mapView)
+
+        // Add labels for the toggles.
+        let museumLabel = UILabel()
+        museumLabel.text = "Show museums"
+        museumLabel.adjustsFontSizeToFitWidth = true
+        museumLabel.backgroundColor = .white
+        museumLabel.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(museumLabel, aboveSubview: mapView)
+
+        let contourLabel = UILabel()
+        contourLabel.text = "Show contours"
+        contourLabel.adjustsFontSizeToFitWidth = true
+        contourLabel.backgroundColor = .white
+        contourLabel.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(contourLabel, aboveSubview: mapView)
+
+        // Layout the switches and labels.
+        NSLayoutConstraint.activate([
+            museumSwitch.topAnchor.constraint(equalTo: mapView.safeAreaLayoutGuide.topAnchor, constant: 20),
+            museumSwitch.leadingAnchor.constraint(equalTo: mapView.safeAreaLayoutGuide.leadingAnchor, constant: 20),
+            museumLabel.widthAnchor.constraint(equalToConstant: 100),
+            museumLabel.leadingAnchor.constraint(equalTo: museumSwitch.trailingAnchor, constant: 10),
+            museumLabel.heightAnchor.constraint(equalTo: museumSwitch.heightAnchor),
```

Comment:
Label can set its own height based on its content, so this constraint is unnecessary